lib/benchee/benchmark.ex

```diff
@@ -149,16 +165,20 @@ defmodule Benchee.Benchmark do
     end
   end
 
-  defp do_benchmark(finish_time, function, input, run_times, n, now)
-  defp do_benchmark(finish_time, _, _, run_times, _n, now)
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, now)
+  defp do_benchmark(finish_time, _, _, run_times, memory_usages, _n, now)
        when now > finish_time do
-    Enum.reverse run_times # restore correct order important for graphing
+    {Enum.reverse(run_times), memory_usages}
   end
-  defp do_benchmark(finish_time, function, input, run_times, n, _now) do
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, _now) do
+    :erlang.garbage_collect()
+    before_run = :erlang.memory(:total)
     run_time = measure_call(function, input, n)
+    after_run = :erlang.memory(:total)
```

Comment:
I think the before/after memory collection could also do nicely as a part of `measure_call` wdyt?